app/src/test/kotlin/com/intive/picover/parties/repository/PartiesRepositoryTest.kt

```diff
@@ -80,5 +94,33 @@ class PartiesRepositoryTest : ShouldSpec(
 				tested.partyById("1").first()
 			}
 		}
+
+		should("add party with correct id") {
+			val childDatabaseReference: DatabaseReference = mockk {
+				every { key } returns newKey
+				coEvery { setValue(newPartyRemote).await() } returns mockk()
+			}
+			every { databaseReference.push() } returns childDatabaseReference
+			every { partyRemote.copy(newKey) } returns newPartyRemote
+
+			val result = tested.addParty(partyRemote)
+
+			coVerify {
+				childDatabaseReference.setValue(newPartyRemote).await()
+			}
+			result.shouldBeSuccess()
+		}
+
+		should("throw an exception when setValue fails") {
+			val errorMessage = "An exception"
```

Comment:
I think the error message checking is not needed. I would go just with Exception(). Also You have only databaseReference.push() mocking for error, instead of databaseReference.push().setValue, according to Your test name :)

Reply:
@kmaslowiec You should test only code, which is relevant for app business logic. In the production code you don't care about the error message, you don't have any logic connected with it so assertion on message is redundant. 
Of course it's ok for learning purposes but keep in mind that in bigger code base it makes your tests more verbose.